src/mococrw/asymmetric_crypto_ctx.h

```diff
@@ -458,6 +458,12 @@ class ECDSASignaturePrivateKeyCtx : public DigestSignatureCtx, public MessageSig
     std::unique_ptr<Impl> _impl;
 };
 
+/** @brief Serialization formats for ECDSA signatures
+ */
+enum class ECSDASignatureFormat {
```

Comment:
Should we add more documentation than just the names of the enum values? A pointer to the relevant standards, maybe?